modules/chronograf-server/README.md

```diff
@@ -0,0 +1,103 @@
+code by adding a `module` configuration and setting its `source` parameter to URL of this folder:
+
+```hcl
+module "chronograf_server" {
+  # TODO: replace <VERSION> with the latest version from the releases page: https://github.com/gruntwork-io/terraform-aws-influx/releases
+  source = "github.com/gruntwork-io/terraform-aws-influx//modules/chronograf-server?ref=<VERSION>"
+
+  # Specify the ID of the Chronograf AMI. You should build this using the scripts in the install-chronograf module.
+  ami_id = "ami-abcd1234"
+  
+  # Configure and start Chronograf during boot. 
+  user_data = <<-EOF
+              #!/bin/bash
+              sudo systemctl start chronograf
+              EOF
+  
+  # ... See variables.tf for the other parameters you must define for the chronograf-server module
+}
+```
+
+Note the following parameters:
+
+* `source`: Use this parameter to specify the URL of the chronograf-server module. The double slash (`//`) is 
+  intentional and required. Terraform uses it to specify subfolders within a Git repo (see [module 
+  sources](https://www.terraform.io/docs/modules/sources.html)). The `ref` parameter specifies a specific Git tag in 
+  this repo. That way, instead of using the latest version of this module from the `master` branch, which 
+  will change every time you run Terraform, you're using a fixed version of the repo.
+
+* `ami_id`: Use this parameter to specify the ID of an Chronograf [Amazon Machine Image 
+  (AMI)](http://docs.aws.amazon.com/AWSEC2/latest/UserGuide/AMIs.html) to deploy on each server in the cluster. You
+  should install Chronograf the scripts in the 
+  [install-chronograf](https://github.com/gruntwork-io/terraform-aws-influx/tree/master/modules/install-chronograf) module.
+  
+* `user_data`: Use this parameter to specify a [User 
+  Data](http://docs.aws.amazon.com/AWSEC2/latest/UserGuide/user-data.html#user-data-shell-scripts) script that each
+  server will run during boot. This is where you can use the 
+  [run-chronograf](https://github.com/gruntwork-io/terraform-aws-influx/tree/master/modules/run-chronograf) 
+  script to configure and run Chronograf as either a meta node or a data node. 
+
+You can find the other parameters in [variables.tf](variables.tf).
+
+Check out the [examples folder](https://github.com/gruntwork-io/terraform-aws-influx/tree/master/examples) for 
+fully-working sample code.
+
+## How do you connect to the Chronograf server?
+
+Once deployed you an simply access the web UI by visiting the public IP of the server on the specified port.
+
+## What's included in this module?
+
+This module creates the following:
+
+* [Auto Scaling Group](#auto-scaling-group)
+* [Security Group](#security-group)
+
+### Security Group
+
+The EC2 Instance has a Security Group that allows minimal connectivity:
+
+* All outbound requests
+* Inbound SSH access from the CIDR blocks and security groups you specify
```

Comment:
What about inbound access to the UI itself? On what port?

Reply:
This module doesn't handle that. It happens in the `chronograf-security-group-rules` module